src/transformers/quantizers/quantizer_torchao.py

```diff
@@ -162,7 +162,7 @@ def _process_model_after_weight_loading(self, model):
 
     @property
     def is_serializable(self):
-        return False
+        return True
```

Comment:
Also we can add a check here to make sure that we have the required version of huggingface-hub to perform serialization.

Reply:
makes sense, I can use `0.25.0.dev0` (my local version), please let me know if there is a better version to check

Reply:
Looks like 0.25 is the version to check against ! The new version was release today: https://github.com/huggingface/huggingface_hub/releases/tag/v0.25.0